test/Test/unit.jl

```diff
@@ -0,0 +1,14 @@
+@testset "Unit Tests" begin
+    mock = MOIU.MockOptimizer(Model{Float64}())
+    config = MOIT.TestConfig()
+    MOIT.unittest(mock, config, [
+        "solve_blank_obj",
+        "solve_constant_obj",
+        "solve_singlevariable_obj",
+        "solve_with_lowerbound",
+        "solve_with_upperbound",
+        "getconstraint",
+        "getvariable",
```

Comment:
`getconstraint`, `getvariable` and `variablenames` should work now that https://github.com/JuliaOpt/MathOptInterface.jl/pull/336 is merged.
You can test the `solve` tests

Reply:
I'm not sure that the mock optimizer implements the names correctly. I got some errors. I'm going to merge this as is, the outstanding tests can be addressed in later PR's.